docs/source/miniconda.rst

```diff
@@ -14,6 +14,21 @@ packages from the Anaconda repository.
 
 `See if Miniconda is right for you <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html#anaconda-or-miniconda>`_.
 
+
+System requirements
+===================
+
+* License: Free use and redistribution under the terms of the `EULA for Anaconda Individual Edition <https://www.anaconda.com/end-user-license-agreement-miniconda>`_. 
+* Operating system: Windows 8 or newer, 64-bit macOS 10.13+, or Linux, including Ubuntu, RedHat, CentOS 7+, and others.
+* If your operating system is older than what is currently supported, you can find older versions of the Anaconda installers in our `archive <https://repo.anaconda.com/archive/>`_ that might work for you. See :ref:`Using Anaconda on older operating systems <old-os>` for version recommendations.
```

Comment:
We need to create a Miniconda equivalent table of what versions of Miniconda stopped supporting which OS.  Until we have it, you need to remove the last sentence.  It is unhelpful to offer Anaconda Individual Edition recommendations for someone looking to install Miniconda.  Also, all Miniconda installers are available here: https://repo.anaconda.com/miniconda/ not in the anaconda installer archive - so that link needs to be updated. as well as the text "Anaconda installers" replaced with Miniconda installers.

Reply:
So I just saw this PR with the latest installers table https://github.com/conda/conda-docs/pull/738, but that doesn't satisfy the need for a table showing Miniconda OS compatibility... who can I speak to about creating a table for that? @csoja 
Thanks